rfai/sql_script/create_table.sql

```diff
@@ -0,0 +1,23 @@
+-- --------------------------------------------------
```

Comment:
remove this quotes add comments if necessary so that anyone can execute the script as it is if needed